prow/plugins/config.go

```diff
@@ -1244,6 +1245,16 @@ func validateTrigger(triggers []Trigger) error {
 	return nil
 }
 
+var warnRepoMilestone time.Time
+
+func validateRepoMilestone(milestones map[string]Milestone) {
+	for _, milestone := range milestones {
+		if milestone.MaintainersID != 0 {
+			logrusutil.ThrottledWarnf(&warnRepoMilestone, time.Hour, "deprecated field: maintainers_id is configured for repo_milestone, maintainers_team should be used instead")
```

Comment:
I'm a little concerned that this warning will pop up too frequently, is this triggered every time a plugin is triggered, or every time prow config is reloaded?

Reply:
oh it's once per hour, good to learn! then it should be fine